src/python/WMCore/WMSpec/Steps/Fetchers/PileupFetcher.py

```diff
@@ -57,6 +58,9 @@ def _queryDbsAndGetPileupConfig(self, stepHelper, dbsReader, fakeSites):
         a subset of the blocks in a dataset will be at a site.
 
         """
+        # only production PhEDEx is connected (This can be moved to init method
+        phedex = PhEDEx()
+        node_filter = set(['UNKNOWN', None])
```

Comment:
I am not sure why but there were cases we got UNKNOWN or None before. (The test I did contain this case though)